pkg/node/devnode.go

```diff
@@ -371,6 +371,12 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 		))
 	)
 
+	var (
+		syncStatusFn = func() (isDone bool, err error) {
```

Comment:
Would be nice if you could put some comment on it, e.g.: "syncStatusFn mocks sync status because...".